pixel-saver@deadalnix.me/buttons.js

```diff
@@ -229,9 +232,28 @@ function init(extensionMeta) {
 	extensionPath = extensionMeta.path;
 }
 
+function loadOptions() {
+	if (options == null) {
+		options = new Settings.Options();
+		options.SHOW_BUTTONS.changed(function() {
+			if (options.SHOW_BUTTONS.get()) {
+				showButtons();
+			} else {
+				hideButtons();
+			}
+		});
+	}
```

Comment:
Maybe we could keep buttons.js free of code messing with options. What do you think about keeping this in settings.js and from there modifying state of buttons module. I've implemented something like that in pull #53 in extension.js enableSettings() function